packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -86,6 +90,14 @@ const getCallEndReason = (call: CallState): CallEndReasons => {
     return CallEndReasons.REMOVED_FROM_CALL;
   }
 
+  if (call.callEndReason?.subCode && call.callEndReason.subCode === ROOM_NOT_FOUND_SUB_CODE) {
```

Comment:
we should conditionally compile these - unless we are *certain* we can't get these subcodes unintentionally